src/components/vulnerabilities/Form.js

```diff
@@ -41,7 +41,17 @@ const VulnerabilityForm = ({
                 const defaultProjectId = projects.length ? projects[0].id : 0;
```

Comment:
This whole logic is super complicated and error-prone. (before you added the changes). We need to find a way to make it simpler in the future